### DISCUSSION TITLE
How to create a Appendix Index List with more than one Column's in Latex/PDF Project's ?

### DISCUSSION THREAD

**Discussion by paule32** [doxygen/doxygen#11204] [Q&A] [answered]
Hello,
like the Title to the Question says:
How to create an Appendix Index List with more than one Column's in Latex/PDF Project's
at the End of Document ?

**Comment by albert-github** [comment, marked as answer]
Doxygen doesn't have a notice of "Appendices" just about chapters and an appendix is "just" another chapter to doxygen.

In the doxygen documentation we use a special `refman.tex` where we introduce "parts" and "Appendices" but no appexdix index.
